chef/cookbooks/network/templates/default/suse-cfg.erb

```diff
@@ -18,40 +18,58 @@ iface=@interfaces[@nic.name]
 -%>
 NAME=<%=quote(@nic.name)%>
 STARTMODE=auto
-<% if iface["slave"] && Nic.bond?(iface["master"])-%>
+<% # For bonding slave we need BOOTPROTO=none to skip the link-layer
+   # setup (see ifcfg(5))
+   if iface["slave"] && Nic.bond?(iface["master"])
+-%>
 BOOTPROTO=none
-<% else -%>
+<% else
+   # All others (including device enslaved into linuxbridge or OVS) get
+   # BOOTPROTO=static
+-%>
 BOOTPROTO=static
 <% end -%>
-<% unless @nic.kind_of?(Nic::Vlan) or @nic.kind_of?(Nic::Bond) or
+<%
+   # ETHTOOL_OPTIONS are only needed for "real" interfaces (i.e.
+   # everything not a bond, bridge or vlan)
+   unless @nic.kind_of?(Nic::Vlan) or @nic.kind_of?(Nic::Bond) or
        @nic.kind_of?(Nic::OvsBridge) -%>
-  <% unless @ethtool_options.empty? -%>
+<%   unless @ethtool_options.empty? -%>
```

Comment:
I am super glad that we add more comments, but I feel this indentation a bit weird (adding spaces inside the <% section)